packages/autorest.python/autorest/codegen/models/combined_type.py

```diff
@@ -79,9 +79,10 @@ def get_json_template_representation(
         client_default_value_declaration: Optional[str] = None,
         description: Optional[str] = None,
     ) -> Any:
-        """Template of what this schema would look like as JSON input"""
-        raise ValueError(
-            "You shouldn't get a JSON template representation of multiple types"
+        return self.types[0].get_json_template_representation(
```

Comment:
Hi @iscai-msft , this is for unbreak generating `Union` property in OpenAI: https://github.com/Azure/azure-rest-api-specs/blob/feature/cognitiveservices/openai-cadl/specification/cognitiveservices/OpenAI.Inference/models/embeddings.create.cadl#L22.

As I understand the `get_json_template_representation` for representing usage example, so I think we can pick any subtype to show in sample? And the description will show "Is one of the following types: ..." to tell the user it's a union.

Reply:
I think this is fine for now, but we might want to revisit this later, right now we're only generating template representation for one type